src/Microsoft.TemplateEngine.Edge/Settings/TemplateCache.cs

```diff
@@ -4,14 +4,17 @@
 using System.IO;
 using Microsoft.TemplateEngine.Abstractions;
 using Microsoft.TemplateEngine.Edge.Template;
-using Microsoft.TemplateEngine.Utils;
 using Newtonsoft.Json;
 using Newtonsoft.Json.Linq;
 
+#nullable enable
+
 namespace Microsoft.TemplateEngine.Edge.Settings
 {
     internal class TemplateCache
     {
+        internal const string CurrentVersion = "1.0.0.5";
```

Comment:
I remember discussing removing it - do we need it?
Each new host version recreates it anyway... or there any use cases of reusing it?

if we need it, should we use `SemanticVersion` or similar class for it?